strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/DefaultExceptionHandler.java

```diff
@@ -87,6 +88,11 @@
         HttpHeaders headers = new HttpHeaders();
         return handleExceptionInternal(ex, body, headers, HttpStatus.SERVICE_UNAVAILABLE, request);
     }
+    
+    @ExceptionHandler(Http202PropogateException.class)
+    protected void handleHttp202PropogateException(Exception ex) {
+        logger.debug(ex.getMessage());        
```

Comment:
@BogdanSukonnov you should provide HTTP 202 response code here as well.

Reply:
done